tree/treeplayer/inc/TTreeReaderValue.h

```diff
@@ -118,8 +118,8 @@ Base class of TTreeReaderValue.
       /// Stringify the template argument.
       static std::string GetElementTypeName(const std::type_info& ti);
 
-      int          fHaveLeaf : 1;                 ///< Whether the data is in a leaf
-      int          fHaveStaticClassOffsets : 1;   ///< Whether !fStaticClassOffsets.empty()
+      int          fHaveLeaf = 1;                 ///< Whether the data is in a leaf
+      int          fHaveStaticClassOffsets = 1;   ///< Whether !fStaticClassOffsets.empty()
```

Comment:
This changes the data definition: `fHaveLeaf : 1` means that it's a one-bit field while `= 1` initializes it to the value `1`. Where exactly is this warning thrown, ie what are we assigning to this field?

Reply:
Thanks for the comment. Yes, you are right, but it's used as an int all over, for example here:
```
root/tree/treeplayer/src/TTreeReaderValue.cxx:379:37: warning: implicit truncation from 'int' to a one-bit wide bit-field changes value from 1 to -1 [-Wsingle-bit-bitfield-constant-conversion]
            fHaveStaticClassOffsets = 1;
                                    ^ ~
```